include/wil/coroutine.h

```diff
@@ -161,6 +161,7 @@
 
 #include <atomic>
 #include <exception>
+#include <utility>

```

Comment:
This was a preexisting problem.  This file uses `std::exchange` but did not include `<utility>`.